web-app/src/components/tournament/details/OutcomeTableItem.vue

```diff
@@ -0,0 +1,216 @@
+<template>
```

Comment:
Name of the file is inappropriate. Should be ResultTableItem. Not Outcome.

Reply:
Wynik, rezultat. This word was used by company in doc

Reply:
Imo "score" its the best